lte/gateway/c/oai/tasks/mme_app/mme_app_procedures.c

```diff
@@ -115,6 +115,76 @@ static void mme_app_free_s11_procedure_create_bearer(
   free_wrapper((void**) s11_proc);
 }
 
+//------------------------------------------------------------------------------
+int mme_app_run_s1ap_procedure_modify_bearer_ind(
+    mme_app_s1ap_proc_modify_bearer_ind_t* proc,
+    const itti_s1ap_e_rab_modification_ind_t* const e_rab_modification_ind) {
+  OAILOG_FUNC_IN(LOG_MME_APP);
+  struct ue_mm_context_s* ue_context_p = NULL;
+  memcpy(
+      (void*) &proc->e_rab_to_be_modified_list,
+      (void*) &e_rab_modification_ind->e_rab_to_be_modified_list,
+      sizeof(proc->e_rab_to_be_modified_list));
+
+  memcpy(
+      (void*) &proc->e_rab_not_to_be_modified_list,
+      (void*) &e_rab_modification_ind->e_rab_not_to_be_modified_list,
+      sizeof(proc->e_rab_not_to_be_modified_list));
+
+  ue_context_p = mme_ue_context_exists_mme_ue_s1ap_id(proc->mme_ue_s1ap_id);
+
+  if (!ue_context_p) {
+    OAILOG_INFO(
+        LOG_MME_APP, "No UE session pool is found" MME_UE_S1AP_ID_FMT ". \n",
+        proc->mme_ue_s1ap_id);
+    OAILOG_FUNC_RETURN(LOG_MME_APP, RETURNerror);
+  }
+  // todo: LOCK_UE_SESSION_POOL
+  // todo: checking on procedures of the function..
+  // mme_app_is_ue_context_clean(ue_context)?!?
+  /** Get the PDN Context. */
+  // TODO sort by pdn_context if different SGWs (not the case now)
+  // pdn_context_t* registered_pdn_ctx =
+  //  RB_MIN(PdnContexts, &ue_context_p->pdn_contexts);
```

Comment:
Remove commented code

Reply:
Comments removed but remarks mentioned TO DO are kept.